### DISCUSSION TITLE
[Q&A] 有办法切回 function_call 的版本吗，好几个代理都没法用 tool_call

### DISCUSSION THREAD

**Discussion by stanfil** [lobehub/lobehub#939] [General Question] [answered]
### 🧐 问题描述 | Proposed Solution

有办法切回 function_call 的版本吗，好几个代理 apikey 都没法用 tool_call

**Comment by arvinxx** [comment, marked as answer]
不计划，建议让代理商兼容 tool calls，毕竟openai 的官方api已经把 function call 废弃了，而且官方的所有老模型也是支持 tool calls 的。

**Comment by stanfil** [comment]
更新：本地 node_module 的问题，删除后使用 pnpm 重装后正常，使用 bun install 无法恢复。

**Reply by arvinxx** [reply]
和这个问题的关系？

**Reply by stanfil** [reply]
之前以为是 apikey 的问题，后面排查发现 curl 直接请求有响应，但是项目代码 chat 接口一直没响应
